recipe/meta.yaml

```diff
@@ -22,44 +22,44 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
-    - numpy  <1.24,>=1.21                    # [build_platform != target_platform]
+    - numpy  >=1.21,<1.24                    # [build_platform != target_platform]
   host:
     - python # >=3.7,<3.10
     - pip
   run:
     - python # >=3.7,<3.10
-    - numpy <1.24,>=1.21
-    - scipy <1.10.0,>=1.5.4
-    - pandas !=1.4.0,<1.6,>=1.2.5
-    - scikit-learn <1.2,>=1.0.0
-    - pillow <=9.4.0,>=9.3.0
+    - numpy >=1.21,<1.24
```

Comment:
Can you remove the upper bound on numpy as well (even though I doubt that something depends on this already).

Reply:
I have removed the numpy upper bound. Still getting the same error. 
> nptyping.error.InvalidArgumentsError: Unexpected argument '(typing.Any,)', expecting Shape[<ShapeExpression>] or Literal[<ShapeExpression>] or typing.Any.